core/src/main/java/hudson/UDPBroadcastThread.java

```diff
@@ -144,4 +203,70 @@ public void shutdown() {
             throw new Error(e);
         }
     }
+    
+    @Extension(ordinal=193) // after DNSMultiCast
+    public static class UDPBroadcastThreadGlobalConfiguration extends GlobalConfiguration {
+        public UDPBroadcastThreadGlobalConfiguration() {
+            load();
+        }
+        
+        // JENKINS-33596 - disable UDP by default
+        private boolean isUdpBroadcastEnabled = false;
+        private int udpBroadcastPort = PORT;
+        
+        @Override
+        public GlobalConfigurationCategory getCategory() {
+            return GlobalConfigurationCategory.get(GlobalConfigurationCategory.Security.class);
+        }
+        
+        public boolean isUdpBroadcastEnabled() {
+            return isUdpBroadcastEnabled;
+        }
+        
+        public void setUdpBroadcastEnabled(boolean isUdpBroadcastEnabled) {
+            this.isUdpBroadcastEnabled = isUdpBroadcastEnabled;
+        }
+        
+        public int getUdpBroadcastPort() {
+            return udpBroadcastPort;
+        }
+        
+        public void setUdpBroadcastPort(int udpBroadcastPort) {
+            this.udpBroadcastPort = udpBroadcastPort;
+        }
+        
+        @Override
+        public boolean configure(StaplerRequest req, JSONObject json) throws FormException {
+            Jenkins jenkins = Jenkins.getInstance();
+            jenkins.checkPermission(Jenkins.ADMINISTER);
+
+            // for compatibility reasons, the actual value is stored in Jenkins
```

Comment:
But the fields were removed from Jenkins, so I guess this information is stored now in its own XML file, no?